cmd/zed/api/cli.go

```diff
@@ -41,123 +40,84 @@ With "zed api" you can create spaces, list spaces, post data to spaces, and run
 The Brim application and the "zed api" client use the same REST API
 for interacting with a zed lake.
 `,
-	New: func(parent charm.Command, flags *flag.FlagSet) (charm.Command, error) {
-		return New(flags)
-	},
+	New: New,
 }
 
-func init() {
-	Cmd.Add(charm.Help)
-}
-
-func New(f *flag.FlagSet) (charm.Command, error) {
-	c := &Command{
-		ctx: newSignalCtx(syscall.SIGINT, syscall.SIGTERM),
-	}
-
+func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
+	c := &Command{Command: parent.(*root.Command)}
 	// If not a terminal make nofancy on by default.
 	c.NoFancy = !terminal.IsTerminalFile(os.Stdout)
-
 	defaultHost := "localhost:9867"
-	f.StringVar(&c.Host, "h", defaultHost, "<host[:port]>")
+	f.StringVar(&c.Host, "service", defaultHost, "<host[:port]>")
```

Comment:
I don't want to make you change all those tests, but I think `-service` is a lot to type for such a common option. Perhaps `-srv`?

Reply:
I'll do -host.  It's an easy find/sed one liner.